src/main/java/edu/hm/hafner/analysis/parser/MsBuildParser.java

```diff
@@ -20,13 +20,12 @@
  */
 public class MsBuildParser extends LookaheadParser {
     private static final long serialVersionUID = -2141974437420906595L;
-
-    private static final String MS_BUILD_WARNING_PATTERN
-            = "(?:^(?:.*)Command line warning ([A-Za-z0-9]+):\\s*(.*)\\s*\\[(.*)\\])|"
-            + ANT_TASK + "(?:(?:\\s*(?:\\d+|\\d+:\\d+)>)?(?:(?:(?:(.*?)\\((\\d*)(?:,(\\d+))?[a-zA-Z]*?\\)|.*LINK)\\s*:|"
-            + "(.*):)\\s*([A-z-_]*\\s(?:[Nn]ote|[Ii]nfo|[Ww]arning|(?:fatal\\s*)?[Ee]rror))[^A-Za-z0-9]\\s*:?\\s*([A-Za-z0-9\\-_]+)?"
-            + "\\s*:\\s(?:\\s*([A-Za-z0-9.]+)\\s*:)?\\s*(.*?)(?: \\[([^\\]]*)[/\\\\][^\\]\\\\]+\\])?"
-            + "|(.*)\\s*:.*error|\\s*warning\\s*(LNK[0-9]+):\\s*(.*)))$";
+        private static final String MS_BUILD_WARNING_PATTERN
```

Comment:
Hey @KalleOlaviNiemitalo 

The groups would not work without a latin letter. So I added one before the naming.

Reply:
The idea behind groups is to use names rather than numbers. Example: https://github.com/jenkinsci/analysis-model/blob/f805dea38333c64cf0caaf8d2e71cc2195c6cf83/src/main/java/edu/hm/hafner/analysis/parser/EmbeddedEngineerParser.java#L27

Reply:
And when you are using groups with names you need to refer to them with the name afterwards. See https://github.com/jenkinsci/analysis-model/blob/f805dea38333c64cf0caaf8d2e71cc2195c6cf83/src/main/java/edu/hm/hafner/analysis/parser/EmbeddedEngineerParser.java#L77

Reply:
And the `:` should not be part of the group